search/indexing_api.py

```diff
@@ -304,6 +322,16 @@ def clear_index():
     create_mappings()
 
 
+def delete_index():
```

Comment:
I thought about this and decided to just copy the code because the `clear_index` will still need `conn` and `index_name`, so either I make `delete_index` return these two variables (but it does not make sense to me, given the scope of `delete_index` or I keep the statements anyway... and at this point I just save 1 line... 
not sure... this is up to you